src/refract/ExpandVisitor.cc

```diff
@@ -194,6 +199,19 @@ namespace refract
                 return ref;
             }
 
+            if (std::find(members.begin(), members.end(), href->value) != members.end()) {
+
+                std::stringstream msg;
+                msg <<  "mixin '";
```

Comment:
I mean, it is good idea to say it is about troubles in mixin.
So I propose something like `named type 'A' is circularly referencing itself by mixin`